fiftyone/utils/annotations.py

```diff
@@ -145,6 +146,12 @@ def annotate(
             one of the supported methods. For existing label fields, if classes
             are not provided by this argument nor ``label_schema``, the
             observed labels on your dataset are used
+        content_type (None): specified the mime type of the provided samples. 
```

Comment:
`Sample.filepath` is always available, so the only reason I could see needing to manually specify `content_type` is if you had a BMP-formatted image stored with a `.jpg` extension for something like that.